zipline/src/commonMain/kotlin/app/cash/zipline/internal/bridge/outbound.kt

```diff
@@ -99,7 +98,7 @@ internal class OutboundCall(
   internal suspend fun <R> invokeSuspending(serializer: KSerializer<R>): R {
     require(callCount++ == parameterCount)
     return suspendCoroutine { continuation ->
-      CoroutineScope(context.endpoint.dispatcher).launch {
+      context.endpoint.scope.launch {
```

Comment:
I am skeptical that this is needed... all we're doing is bookkeeping and sending notifications.

Reply:
I was specifically trying to handle the caller calling us on the wrong dispatcher. We should either thread-hop or detect mistakes.

Reply:
Hmm yeah I guess that can happen from JVM. I was thinking in the context of JS calling out.